tests/lib/rules/use-v-on-exact.js

```diff
@@ -59,17 +74,195 @@ ruleTester.run('use-v-on-exact', rule, {
+    },
+    {
+      code: `<template><input @keypress.exact="foo" @keypress.middle="bar" @keypress.ctrl="baz"></template>`
+    },
+    {
+      code: `<template><input @keypress.middle="foo" @keypress.middle="bar"></template>`
+    },
+    {
+      code: `<template><UiButton @click="foo" /></template>`
+    },
+    {
+      code: `<template><UiButton @click="foo" @click.native="bar" /></template>`
+    },
+    {
+      code: `<template><UiButton @click="foo" @click.native.ctrl="bar" /></template>`
+    },
+    {
+      code: `<template><UiButton @click="foo" @click.native.exact="bar" @click.native.ctrl="baz" /></template>`
+    },
+    {
+      code: `<template><UiButton @click.native.exact="bar" @click.ctrl.native="baz" /></template>`
+    },
+    {
+      code: `<template><UiButton @click.native.ctrl.exact="foo" @click.native.ctrl.shift="bar" /></template>`
+    },
+    {
+      code: `<template><UiButton @click.native="foo" @click.native.stop="bar" /></template>`
+    },
+    {
+      code: `<template><UiButton @click.native.stop="foo" @click.native.prevent="bar" /></template>`
     }
   ],
 
   invalid: [
     {
-      code: `<template><button @click="foo" @click.ctrl="foo"/></template>`,
-      errors: ["Consider to use '.exact' modifier."]
+      code: `<template>
+        <button
+          @click="foo"
+          @click.ctrl="bar"
+        />
+      </template>`,
+      errors: [
+        { message: "Consider to use '.exact' modifier.", line: 3 }
+      ]
+    },
+    {
+      code: `<template>
+        <button
+          @click="foo"
+          @click.ctrl.stop="bar"
+        />
+      </template>`,
+      errors: [
+        { message: "Consider to use '.exact' modifier.", line: 3 }
+      ]
+    },
+    {
+      code: `<template>
+        <button
+          @click.prevent="foo"
+          @click.ctrl="bar"
+        />
+      </template>`,
+      errors: [
+        { message: "Consider to use '.exact' modifier.", line: 3 }
+      ]
+    },
+    {
+      code: `<template>
+        <button
+          @click.exact="foo"
+          @click.ctrl="bar"
+          @click.ctrl.shift="baz"
+        />
+      </template>`,
+      errors: [
+        { message: "Consider to use '.exact' modifier.", line: 4 }
+      ]
+    },
+    {
+      code: `<template>
+        <button
+          @click="foo"
+          @click.ctrl="bar"
+          @click.ctrl.shift="baz"
+        />
+      </template>`,
+      errors: [
+        { message: "Consider to use '.exact' modifier.", line: 3 },
+        { message: "Consider to use '.exact' modifier.", line: 4 }
+      ]
+    },
+    {
+      code: `<template>
+        <input
+          @keypress.27="foo"
+          @keypress.27.middle
+        >
+      </template>`,
+      errors: [
+        { message: "Consider to use '.exact' modifier.", line: 3 }
+      ]
+    },
+    {
+      code: `<template>
+        <input
+          @keydown.a.b.c="abc"
+          @keydown.a="a"
```

Comment:
If I understand correctly - then we'd have to disable this rule for the specific case. Or you do something like this instead:
```
@keydown.a.exact="a"
@keydown.a.b.exact="ab"
@keydown.a.b.c="abc"
```

Reply:
Forget what I said, it wasn't clear and on top of that I was wrong. The exact modifier only makes sense with system modifiers, not with regular keys. So this one shouldn't throw any error

Reply:
Oh damn. I missed it! I was expecting that the following code: `@keydown.a.b="foo"` would be executed while `a` and `b` are both pressed, but nope - it will fire for both a and b, no matter `exact` modifier presence. Thanks for bringing it up @posva 🖖